recipe/meta.yaml

```diff
@@ -34,6 +36,9 @@ requirements:
     - {{ pin_compatible('numpy') }}
     - snuggs >=1.4.1
     - click-plugins
+    # not an explicit dependency but tests clearly depend on proj version
+    # test_target_aligned_pixels fails for proj 8.0.0
+    - proj
```

Comment:
@conda-forge/rasterio, please let me know if you're okay with this addition.  Without it, we are pulling in `proj 8.0.0` (via the latest `libgdal` and a test (`test_target_aligned_pixels`) is failing.

Reply:
I might need to (or want to) add the `proj 7.2` migrator.